src/Components/TriggerInfo/TriggerInfo.jsx

```diff
@@ -61,9 +61,14 @@ function ScheduleView(props: { data: Schedule }): React.Node {
         .format("HH:mm")}–${moment("1900-01-01 00:00:00")
         .add(endOffset, "minutes")
         .format("HH:mm")}`;
+
+    const timeZone = "GMT " + (tzOffset < 0 ? "- " : "+ ") +
+        `0${Math.abs(tzOffset) / 60 ^ 0}`.slice(-2) + ':' +
+        ('0' + Math.abs(tzOffset) % 60).slice(-2);
```

Comment:
In my opinion this variable declaration is overloaded and not clear enough. Let's divide it to different operations like calculation of hours, minutes and definition of plus or minus sign

Reply:
Also we already have moment.js in dependencies. @prateek2211 You can try to use it to get GMT